src/librustc_metadata/cstore.rs

```diff
@@ -244,6 +247,52 @@ impl CStore {
     {
         self.extern_mod_crate_map.borrow().get(&emod_id).cloned()
     }
+
+    pub fn visible_parent_map<'a>(&'a self) -> ::std::cell::RefMut<'a, DefIdMap<DefId>> {
```

Comment:
The `RefCell` has to be borrowed mutably in this function and I didn't want to needlessly re-borrow it immutably before returning. This function is only used in `librustc_metadata` so I don't think it makes much of a difference.